inst/pages/21_microbiome_community.qmd

```diff
@@ -1,3 +1,9 @@
+---
+editor: 
+  markdown: 
+    wrap: 72
+---
```

Comment:
What these does?

I think these options should be applied to whole book (there should not be unique options for single pages (makes the maintenance a lot harder)

Reply:
Check if other files have this. If not, then I think we could remove this. You could also check what is the default option in OMA currently

I think this controls how long lines are wrapped

Reply:
yeah it was an anomaly removed it